config/katello-answers.yaml

```diff
@@ -1,41 +1,54 @@
+# Format:
+# <classname>: false - don't include this class
+# <classname>: true - include and use the defaults
+# <classname>:
+#   <param>: <value> - include and override the default(s)
+#
+# See params.pp in each class for what options are available
+
 ---
 certs:
   group: foreman
 katello: true
+global: true
+application: true
+pulp: true
+candlepin: true
```

Comment:
This fits nicely with where the new certificate work is headed as it would allow users to configure the certificates if they wanted and set the defaults, e.g. https://github.com/theforeman/puppet-katello/pull/351/files#diff-0a129cc5a540f220caade2da4fc3229b